ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -77,5 +77,45 @@
         </nhs-form-group>
     </validate-input>
     <br />
+
+    <h2>Notification Date</h2>
+    <validate-date model="Notification" property="NotificationDate" notification_id="@Model.Notification.NotificationId" inline-template>
+        @{
+            var NotificationDateError = !Model.IsValid("Notification.NotificationDate");

```

Comment:
Convention for local variables to be camel-case (lower first) but not majorly fussed here.